app/controllers/api/v0/profile_images_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Api
+  module V0
+    class ProfileImagesController < ApiController
+      before_action :set_cache_control_headers, only: %i[show]
```

Comment:
If you set the caching headers, you also need a way to destroy that cache, so you need the surrogate keys for the resource, which in this case it could be a user or an org.

I'm not sure it's worth to cache this endpoint though and in general I think we should lean towards caching endpoints that have either complex fetching queries or complex serialization structures.

This have neither, so I'd lean towards removing the cache altogether

Reply:
Yup, didn't looked into this with much attention - giving the method a quick glance I got the idea that it was just setting a 1 day cache control header (I didn't even realize that there was a cache invalidation mechanism also put in place).

Thanks for noticing this, I've removed the action. 👏